src/config/dynamic-theme-fixes.config

```diff
@@ -12078,6 +12078,14 @@ body, img, video, iframe {
 
 ================================
 
+pipiwiki.com
+
+INVERT
+.mwe-math-fallback-image-inline
```

Comment:
You might want to do:

```css
.mwe-math-fallback-image-inline
.mwe-math-fallback-image-display
```

This is also how we fix these equations on wikipedia.org